src/app/screens/connect/btcSelectAddressScreen/index.tsx

```diff
@@ -0,0 +1,292 @@
+import BitcoinIcon from '@assets/img/dashboard/bitcoin_icon.svg';
+import stxIcon from '@assets/img/ledger/stx_icon.svg';
+import OrdinalsIcon from '@assets/img/nftDashboard/white_ordinals_icon.svg';
+import ActionButton from '@components/button';
+import SelectAccount from '@components/selectAccount';
+import useBtcAddressRequest from '@hooks/useBtcAddressRequest';
+import useWalletSelector from '@hooks/useWalletSelector';
+import { Check } from '@phosphor-icons/react';
+import { animated, useSpring } from '@react-spring/web';
+import { StickyHorizontalSplitButtonContainer } from '@ui-library/common.styled';
+import { getTruncatedAddress } from '@utils/helper';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { AddressPurpose } from 'sats-connect';
+import styled, { useTheme } from 'styled-components';
+import { getAppIconFromWebManifest } from './helper';
+
+const TitleContainer = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  overflow: 'hidden',
+  marginLeft: 30,
+  marginRight: 30,
+  marginTop: 48,
+});
+
+const OuterContainer = styled(animated.div)({
+  display: 'flex',
+  flexDirection: 'column',
+  marginLeft: 16,
+  marginRight: 16,
+});
```

Comment:
actual:
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/97f6b7e0-baa4-43d0-b2af-2f361c0042f5)

figma:
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/8d4fcd84-4e7e-46b1-8b64-4208839a96cb)

missing scrollbar on hover. use the one provided in theme.ts